requirements.txt

```diff
@@ -1,2 +1,34 @@
 numpy
-pytest
\ No newline at end of file
+pytest
```

Comment:
Numpy is in requirements.txt but NOT in environment.yml. Which one is correct? Do you need both files?

Reply:
I don't think numpy is needed, so it will be removed. I'm not sure if both files are needed right now. I have gotten the CI into a stable configuration, but definitely not the final one. Things to be done in the CI include installing Cyclus and Cycamore from branches off my forks to use the most updated versions, instead of an out-of-date docker image (or updating the docker image). One I update the builds to the most current versions, I will determine if both of these files are needed.